packages/editor/src/client/locales/en.json

```diff
@@ -118,7 +118,8 @@
         "hideAuthors": "Hide Authors",
         "socialMediaImage": "Image for Social Media",
         "postImage": "Image for Post",
-        "metadataInfo": "Information from these fields are being used for previews in Social Media Posts. If left blank, information from general-tab will be used instead."
+        "metadataInfo": "Information from these fields are being used for previews in Social Media Posts. If left blank, information from general-tab will be used instead.",
+        "lightStyledWithNoImageError": "Light styled teaser must has an image"
```

Comment:
"Light styled teasers must have an image".

Also the location of the translation is wrong. You're in the `article` translations and i think this should go to the `pages` translations.

Reply:
I guess in both article and page?

Reply:
@roosnic1 I added it in both article and page panels, tell me what you think